src/support.js

```diff
@@ -436,7 +436,10 @@ function loadWebAssemblyModule(binary, flags) {
     // b) Symbols from loaded modules are not always added to the global Module.
     var moduleLocal = {};
 
-    var resolveSymbol = function(sym, type) {
+    var resolveSymbol = function(sym, type, legalized) {
+      if (legalized) {
+        sym = 'orig$' + sym;
+      }
 #if WASM_BACKEND
       sym = '_' + sym;
```

Comment:
Should we use asmjs_mangle here?  Also does the mangling happen before the `orig$` .. i.e. is it `_orig$foo` or `orig$foo`.   Above it looks like your remove the mangled before pretending `orig$`.

Reply:
This is pre-existing code, but I think `asm_mangle` in python makes sense (where it checks if the `_` is needed or not, which it isn't for `dynCall` etc.) while here, all we have are exported symbols at the C level, which does not include things like `dynCall` etc. that are only for JS.